jobs/JREGIONAL_GET_EXTRN_MDL_FILES

```
@@ -79,7 +79,10 @@ to generate initial or lateral boundary conditions for the FV3 model.
 #-----------------------------------------------------------------------
 #
 valid_vals_ICS_OR_LBCS=( "ICS" "LBCS" )
-check_var_valid_value "ICS_OR_LBCS" "valid_vals_ICS_OR_LBCS"
+ICS_OR_LBCS=${ICS_OR_LBCS:-""}
+if [ ! "${ICS_OR_LBCS}" = "" ]; then
+  check_var_valid_value "ICS_OR_LBCS" "valid_vals_ICS_OR_LBCS"
```

@lgannoaa, why is this if-statement necessary?

---

During some test case, we found the job silently failed due to ICS_OR_LBCS not eval at the first before invoke check_var_valid_value. Therefore, this exception handling code was added.